pennylane/pulse/parametrized_hamiltonian.py

```diff
@@ -314,7 +315,8 @@ def __radd__(self, H):
         coeffs = self.coeffs.copy()
 
         if isinstance(H, (Hamiltonian, ParametrizedHamiltonian)):
-            new_coeffs = H.coeffs.copy() + coeffs
+            # if Hamiltonian, coeffs array must be converted to list
+            new_coeffs = list(H.coeffs.copy()) + coeffs
```

Comment:
Should we maybe generally enforce lists in the `__add__` methods? 🤔

Reply:
(i.e. also in RydbergHamiltonians)

Reply:
This is a good point. The corresponding things for `RydbergHamiltonian` are added in the [callable phase PR](https://github.com/PennyLaneAI/pennylane/pull/3866), so I will add that there along with a couple of tests!